### ISSUE TITLE
Swipe to dismiss on StickyHeaderListView

### ISSUE THREAD

**Issue by AndyOHart** [nhaarman/ListViewAnimations#341]
Just wondering if this is possible?
I want to have both sticky headers and the swipe to dismiss/undo functionality.
Thanks

**Comment by anzaidemirzoi** [comment]
Yeah I wonder this too, I also want to use starting animation too. is it possible?

**Comment by mklimek** [comment]
+1
It's possible to achieve it right now?

**Comment by LonerganOnline** [comment]
Has anybody found a way of achieving this?
